### DISCUSSION TITLE
Add a simultaneous sending limit per ISP & make script in Postal

### DISCUSSION THREAD

**Discussion by Orminor77** [postalserver/postal#2017] [Help with using Postal]
Hello !

I have two questions please,

1) I would like for example that 3 mails max simultaneous are sent to Yahoo and Gmail 4 mails max simultaneous, how could I do ?
(I ask this because I am blocked because there are too many requests simultaneously on some ISP)

2) Could I use for example a script to do the limitation by ISP? 
(I show you an example of someone who made a script for held messages: https://github.com/postalserver/postal/issues/144#issuecomment-498635520)

Thanks you in advance for your help, if someone has a solution it would be great.

**Comment by willpower232** [comment]
This is not currently a feature of Postal so if you want to slow down the sending, you should do that in whatever application you're using to send the messages.

**Reply by Orminor77** [reply]
Ok thanks you anyway, have a nice day ! 💪
